tests/test_sirx.py

```diff
@@ -0,0 +1,126 @@
+# pylint: disable=C0114
+# pylint: disable=C0116
+# pylint: disable=R0124
+# pylint: disable=R0201
+# pylint: disable=C0121
+"Test exponential convergence"
+import numpy as np
+import pytest
+
+from opensir.models import SIRX
+
+DEFAULT_PARAMS = [
+    6.2 / 8,
+    1 / 8,
+    0.05,
+    0.05,
+    5,
+]  # Default parameters from Maier et al (2020)
+GUANGDONG_IC = [104299972, 14, 0, 14]  # Ealing initial conditions
+
+
+class TestSirx:
+    """ Test class for the SIRX model """
+
+    @pytest.fixture
+    def model(self):
+        return SIRX()
+
+    def test_guangdong(self, model):
+        # Check that guangdong 22 days prediction
+        # is accurate with validated test case
+        t = 22  # 23 days
+        model.set_params(DEFAULT_PARAMS, GUANGDONG_IC)
+        model.solve(t, t + 1)
+        infected_end = model.fetch()[-1, 4]
+
+        assert abs(infected_end - 2524.388983) < 0.0001
+
+    def test_expconv(self, model):
+        # Check that the numerical solution of the
+        # reduction from sir-x to sir
+        # converges to I ~ I_0 * exp (alpha-beta)
+        # as (S,t) -> (1,0)
+        # Asume small amount of infected
+        n_i_0 = 1
+        # Asume large population
+        initial_conds = [GUANGDONG_IC[0], n_i_0, 0, n_i_0]
+        # Asume very short time
+        t = 1 / (24 * 3600)
+        # Make kappa, kappa_0 and
```

Comment:
That final "and" is a bit strange, or is it a "connector" with "Calculate analytical..."?

Reply:
Well spotted! the full string was:

        # Make kappa, kappa_0 and inf_over_test zero to reduce
        # the SIR-X to SIR
But for some reason my brain got disconnected halfway through